PRCS/prc-1.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Paima Achievement Interface
+description: Interface for sharing in-game achievements.
+author: seba@paimastudios.com, edward@paimastudios.com
+status: open
+created: 2023-11-08
+---
+
+## Abstract
+
+A open cross-game standard achievement specification to gamify onchain participation. 
+This specification allows to interact through standard http methods.  
+
+## Motivation
+
+Most games have a achievement system, but there are not compatible: vendors and tools have adapt to each game. By implementing this standard, developer spend no or less time implementing the format itself, vendors can use the format for gamification apps, explorers, prizes, etc.
+
+## Format
+
+### HTTP
+* Network calls are done to any game node via HTTP to `BASE_URL`
+* All network requests are `Method GET`
+* All responses ContentType is `application/json`
+* `Standard HTTP codes` are used for status. E.g., 200 OK, 500 Internal Server Error, 404 Not Found, etc.  
+
+### Response Interfaces
+
+General Game Info
+```
+interface Game = {
+  id: string             // Game ID
+  name?: string          // Optional Game Name
+  version?: string       // Optional Game Version
+}
+```
+
+Data Validity
+```
+interface Validity = {
+  block: number;         // Data Block height (0 always valid)
+  chainId: number;       // Data ChainId
```

Comment:
chainID are not universally numbers. Multiple chains use strings as ids. It's better to use a chain-agnostic standard for defining chain IDs like [CAIP-2](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-2.md)

Reply:
updated field